array-loops.js

```diff
@@ -1,33 +1,33 @@
 // --------------------------
 // Opdracht 1
 // --------------------------
-
 const names = ["Henk", "Piet", "Fred", "Joop"];
-
+for (let i=0; i<names.length; i++) {
+    names[i] += "je";
```

Comment:
Hoi, Ik heb hier gewoon names[i]= names[i] + "je" van gemaakt, vindt ik duidelijker, maar of het beter/slechter is??

Reply:
In deze regel mis je nog een spatie.